config/samples/nfd.openshift.io_v1_nodefeaturediscovery.yaml

```diff
@@ -11,8 +11,8 @@ spec:
   #resourceLabels:
   #  - "example.com/resource"
   operand:
-    image: quay.io/openshift/origin-node-feature-discovery:4.11
-    imagePullPolicy: IfNotPresent
+    image: quay.io/eduardoarango/node-feature-discovery:v0.11.1
```

Comment:
Do we want to point here for downstream?

Reply:
ups....